Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -158,20 +160,33 @@ const Contest = () => {
         [ participantsStateText, participantsValue ],
     );
 
-    const renderErrorMessage = useCallback(() => (
-        <div className={styles.headingContest}>
-            <Heading
-              type={HeadingType.primary}
-              className={styles.contestHeading}
-            >
-                {contestTitle}
-                {' '}
-                -
-                {' '}
-                {validationResult.message}
-            </Heading>
-        </div>
-    ), [ validationResult, contestTitle ]);
+    const renderErrorHeading = useCallback(
+        (message: string) => (
+            <div className={styles.headingContest}>
+                <Heading
+                  type={HeadingType.primary}
+                  className={styles.contestHeading}
+                >
+                    {isNil(contestTitle)
+                        ? ''
+                        : `${contestTitle} - ${message}`}
+                </Heading>
+            </div>
+        ),
+        [ contestTitle ],
+    );
+
+    const renderErrorMessage = useCallback(() => {
+        const { isValid, message: contestErrorMessage } = contestValidation;
+
+        if (!isValid) {
+            return renderErrorHeading(contestErrorMessage);
+        }
+
+        const { message: submissionResultsErrorMessage } = submissionResultsValidation;
+
+        return renderErrorHeading(submissionResultsErrorMessage);
+    }, [ renderErrorHeading, contestValidation, submissionResultsValidation ]);
```

Comment:
In order to follow conventions, place the dependency brackets on new line.

Reply:
Done.